test/Microsoft.AspNetCore.SystemWebAdapters.Tests/Caching/CacheTests.cs

```diff
@@ -103,6 +103,22 @@ public void InsertNoCallbacks()
         memCache.Verify(m => m.Set(key, item, It.Is<CacheItemPolicy>(e => e.AbsoluteExpiration.Equals(absoluteExpriration) && e.SlidingExpiration.Equals(slidingExpiration)), null), Times.Once);
     }
 
+    [Fact]
+    public void InsertNoAbsoluteSlidingExpiration()
+    {
+        // Arrange
+        var memCache = new Mock<MemoryCache>(_fixture.Create<string>(), null);
```

Comment:
What about non-null dependencies?

Reply:
CacheDependency cannot be null in this case, just fixed the method signature.